armi/reactor/converters/uniformMesh.py

```diff
@@ -83,17 +85,29 @@ def __init__(self, cs=None):
         self.blockDetailedAxialExpansionParamNames = []
         self.reactorParamNames = []
 
+        # These dictionaries represent back-up data from the source reactor
+        # that can be recovered if the data is not being brought back from
+        # the uniform mesh reactor when ``applyStateToOriginal`` to called.
+        # This prevents clearing out data on the original reactor that should
+        # be preserved since no changes were applied.
```

Comment:
Is there any way to shorten these comments?

People who are stuck trying to fix/write code have to read a lot of stuff really fast, and this will slow people down significantly if they need look at the `uniformMesh` code at all.